src/app/app.component.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { Location } from '@angular/common';
+import {
+  Component,
+  NO_ERRORS_SCHEMA
+} from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { createMock } from 'testing/mock';
+import {
+  initContext,
+  TestContext
+} from 'testing/test-context';
+
+import { Broadcaster } from 'ngx-base';
+import { BsModalService } from 'ngx-bootstrap/modal';
+import { Spaces } from 'ngx-fabric8-wit';
+import { AuthenticationService } from 'ngx-login-client';
+import {
+  Observable,
+  Subject
+} from 'rxjs';
+
+import { OnLogin } from '../a-runtime-console/index';
```

Comment:
this can be shortened to just `import { OnLogin } from './a-runtime-console/index';`